connect-examples/server-library-example/resources/dummy_data.custom

```diff
@@ -0,0 +1,6 @@
+id,name
+1,John Doe
+2,Jane Smith
+3,Bob Johnson
+4,Alice Williams
+5,Charlie Brown
```

Comment:
Shall we use simply `data.csv` as a file name because this seems to be technically CSV format?

Reply:
Changed it to `.data`! Even though it is technically CSV format, I did not want to use it explicitly in order to show that format can "truly" be customised